aspnetcore/publishing/iis.md

```diff
@@ -34,29 +34,33 @@ Enable the **Web Server (IIS)** server role and establish role services.
 
 Navigate to **Control Panel > Programs > Programs and Features > Turn Windows features on or off** (left side of the screen). Open the group for **Internet Information Services** and **Web Management Tools**. Check the box for **IIS Management Console**. Check the box for **World Wide Web Services**. Accept the default features for **World Wide Web Services** or customize the IIS features to suit your needs.
 
-![image](iis/_static/wf.png)
+![IIS Management Console and World Wide Web Services are selected in Windows Features.](iis/_static/windows-features-win10.png)
 
 ### Windows Server operating systems
 
-For server operating systems, use the **Add Roles and Features Wizard** via the **Manage** menu or the link in **Server Manager**. On the **Server Roles** step, check the box for **Web Server (IIS)**.
+For server operating systems, use the **Add Roles and Features** wizard via the **Manage** menu or the link in **Server Manager**. On the **Server Roles** step, check the box for **Web Server (IIS)**.
 
-![image](iis/_static/rs.png)
+![The Web Server IIS role is selected in the Select server roles step.](iis/_static/server-roles-ws2016.png)
 
 On the **Role services** step, select the IIS role services you desire or accept the default role services provided.
 
-![image](iis/_static/role-services.png)
+![The default role services are selected in the Select role services step.](iis/_static/role-services-ws2016.png)
 
-Proceed through the **Confirmation** step to enable the web server role and services.
-
-<a name=unlock-handlers></a>
+Proceed through the **Confirmation** step to install the web server role and services. A server/IIS restart is not required after installing the Web Server (IIS) role. We will execute a server/IIS restart after installing the .NET Core Windows Server Hosting bundle in the next step.
 
 ## Install the .NET Core Windows Server Hosting bundle
 
 1. Install the [.NET Core Windows Server Hosting](https://aka.ms/dotnetcore_windowshosting_1_1_0) bundle on the server. The bundle will install the .NET Core Runtime, .NET Core Library, and the ASP.NET Core Module. The module creates the reverse-proxy between IIS and the Kestrel server.
 
-2. Restart the server or execute **net stop was /y** followed by **net start w3svc** from the command-line to pickup changes to the system PATH.
+2. Restart the server or execute **net stop was /y** followed by **net start w3svc** from a command prompt to pickup changes to the system PATH.
+
+> [!NOTE]
+> If you only plan to host self-contained applications and thus don't require the .NET Core runtime on the server, you have the option of only installing the ASP.NET Core Module by running the installer from an Administrator command prompt: **DotNetCore.1.1.0-WindowsHosting.exe OPT_INSTALL_LTS_REDIST=0 OPT_INSTALL_FTS_REDIST=0**
 
-For more information, see [ASP.NET Core Module](../fundamentals/servers/aspnet-core-module.md).
+> [!NOTE]
```

Comment:
The reason I have this in a note and play it down by providing it at the end is that this is not the recommended way to host apps. The overwhelming advice from MS contributors in Hosting and IIS repos and on Slack (and JabbR prior) has been to install the full hosting bundle. The self-contained approach just hasn't received the :heart: from MS devs. AFAICT the push for portable was to simplify the experience overall. Concerns have also been raised over deployment size and resource use on servers. For example, my testselfcontained app is a meaty 54MB. The same app in portable form comes in at a sweet 16MB. 

I was attempting to keep the MS focus in mind by making this note parenthetical. In spite of the MS effort to downplay the self-contained (and module-only) approach, I felt that totally ignoring the option isn't so good either. I love self-contained apps. Chats at Slack with devs also indicate a number of others who enjoy deploying their apps in this format.

I guess they'll let me/us know:
1. Move this up, perhaps into step 1, where it pertains to how the hosting bundle installer is run.
2. Strike it completely from the doc, which would make it fairly opaque as an option for hosting apps.
3. Leave it as a parenthetical note (as it appears in the commit) at the bottom so that its not emphasized but also at least brought into awareness.

cc/ @Rick-Anderson @danroth27